lib/merge_supporters.rb

```diff
@@ -66,9 +71,21 @@ def self.selected(merged_data, supporter_ids,np_id, profile_id)
     return {json: new_supporter, status: :ok}
   end
 
+  def self.conflicting_custom_fields?(supporters)
+    cfjs = []
+    supporters.each do |supporter|
+      supporter.custom_field_joins.each do |cfj|
+        cfjs << { 'custom_field_master_id' => cfj.custom_field_master_id, 'value' => cfj.value }
+      end
+    end
+
+    cfjs.group_by{|i| i['custom_field_master_id']}.any?{ |id, group| group.uniq.size > 1 }
+  end
+
 
   # Merge supporters for a nonprofit based on an array of groups of ids, generated from QuerySupporters.dupes_on_email or dupes_on_names
-  def self.merge_by_id_groups(np_id, arr_of_ids, profile_id)
+  def self.merge_by_id_groups(np_id, arr_of_ids, profile_id, skip_conflicting_custom_fields=false)
+    conflicting_custom_fields = []
```

Comment:
Is this the list of supporter ids with conflicts?

Reply:
Yes. I think a better name could be `supporter_ids_with_conflicting_custom_fields`.